src/tokens/ERC20RebaseDistributor.sol

```diff
@@ -197,13 +220,14 @@ abstract contract ERC20RebaseDistributor is ERC20 {
     function balanceOf(
         address account
     ) public view virtual override returns (uint256) {
-        uint256 _rebasingShares = rebasingState[account].nShares;
-        if (_rebasingShares == 0) {
+        RebasingState memory _rebasingState = rebasingState[account];
+        if (_rebasingState.isRebasing == 0) {
             return ERC20.balanceOf(account);
         } else {
-            return
-                (_rebasingShares * rebasingSharePrice) /
+            uint256 underlying = (_rebasingState.nShares * rebasingSharePrice) /
                 START_REBASING_SHARE_PRICE;
+            uint256 rawBalance = ERC20.balanceOf(account);
+            return underlying > rawBalance ? underlying : rawBalance;
```

Comment:
I'd prefer Math.max here to a ternary

Reply:
I'd rather not import a new lib for this